skip tests - Update readme's again

andy7i opened the pull request:
Ticket: <Link to Jira ticket> (none)
<Put notes here to help reviewer understand this PR>
Readme's were still missing some info
---

A couple follow-on tickets:

Make it easier to run Sam locally and run unit tests https://broadworkbench.atlassian.net/browse/CA-809

Make it easier to run integration tests against a local Sam (or other services) https://broadworkbench.atlassian.net/browse/CA-810


---
**PR checklist**

- [x] I've followed [the instructions](https://github.com/broadinstitute/sam/blob/develop/CONTRIBUTING.md#api-changes) if I've made any changes to the API, _especially_ if they're breaking changes
- [x] I've updated the RC_XXX release ticket with any manual steps required to release this change
- [x] I've updated the [FISMA documentation](https://github.com/broadinstitute/sam/blob/develop/CONTRIBUTING.md#fisma-documentation-changes) if I've made any security-related changes, including auth, encryption, or auditing

andy7i committed:
update integration test readme:
- intellij setup
- instructions for running tests in intellij
- config setup

todo: we may want to update the ctmpl (https://github.com/broadinstitute/firecloud-automated-testing/blob/master/configs/sam/application.conf.ctmpl) so that we don't need to add local Sam manually. This is because the scripts support local UI (with a FIAB backend) but not a local service. This affects all services afaik, including rawls, leo, etc. -> create ticket

andy7i committed:
- instructions on how to run tests in intellij
- make setup instructions easier to copy paste in batch
- link to integration test readme

andy7i committed:
make codacy happy

gpolumbo-broad commented on README.md:
```diff
@@ -141,34 +141,37 @@ Build jar and docker image:
-Stop your local opendj:
+
+#### Run tests in intellij
+In `Run` -> `Edit Configurations...`, make sure the ScalaTest template has `VM Parameters` filled in with:
```
I would emphasize that it's the _**template**_ that you're modifying here.  Rewording here slightly to maybe add a sub-heading or a bullut before these instructions to emphasize that it's the template you want to edit.  Or maybe an extra sentence describing how to navigate to the list of templates and find the right ScalaTest one to modify.

gpolumbo-broad commented on automation/README.md:
````diff
@@ -9,35 +9,39 @@ See [firecloud-automated-testing](https://github.com/broadinstitute/firecloud-au
+LOCAL_UI=true ./render-local-env.sh
+```
+
+##### Against local Sam pointing to the dev live env
````
I don't understand this one?  "Local Sam pointing to live dev Sam"?  Do you mean you're running the Sam app locally but using the live dev Sam's database?

andy7i replied:
I initially was thinking of it as local Sam pointing to the live dev Sam's DB, but it should actually just say a local Sam, because the local Sam can point to either dev DB or a local DB. Thanks for the catch -- updated.

gpolumbo-broad commented on automation/README.md:
```diff
@@ -9,35 +9,39 @@ See [firecloud-automated-testing](https://github.com/broadinstitute/firecloud-au
+All test code lives in `automation/src/test/scala`.
 
 #### From IntelliJ
+It is recommended to `Open...` IntelliJ to the `automation` folder directly and import from SBT.
```
nit: I would suggest that a better way to say this is  to open create a new IntelliJ project with just the automation tests in it.

andy7i replied:
I gave it another go. Hopefully my end-of-day brain made some sense🤞

gpolumbo-broad commented on automation/README.md:
```diff
@@ -9,35 +9,39 @@ See [firecloud-automated-testing](https://github.com/broadinstitute/firecloud-au
+It is recommended to `Open...` IntelliJ to the `automation` folder directly and import from SBT.
 
-First, you need to set some default VM parameters for ScalaTest run configurations. In IntelliJ, go to `Run` > `Edit Configurations...`, select `ScalaTest` under `Defaults`, and add these VM parameters:
+First, you need to set some default VM parameters for ScalaTest run configurations. In IntelliJ, go to `Run` > `Edit Configurations...`, select `ScalaTest` under `Templates`, and add these VM parameters:
```
These instructions I think are better than the wording in the other README for how to modify the template

gpolumbo-broad reviewed (changes requested):
I left a few comments, but I really don't feel a need to block the PR on any of them except the one where I said I didn't understand the meaning of the statement.  

Good clarifications though all around, I like it!

andy7i committed:
addressing PR feedback

andy7i committed:
remove fiab info from local sam instructions

a local sam points to a local db or to dev, so the instructions explain how to point to dev as well.

andy7i committed:
make codacy happy

andy7i commented:
got the ok from doge to merge
